polaris-tokens/scripts/index.js

```diff
@@ -0,0 +1,8 @@
+const {osColorSchemes, tokens} = require('../dist/index');
+
+const {toJSON} = require('./toJSON');
+const {toStyleSheet} = require('./toStyleSheet');
+
+(async () => {
+  await Promise.all([toJSON(tokens), toStyleSheet(tokens, osColorSchemes)]);
+})();
```

Comment:
```suggestion
await Promise.all([toJSON(tokens), toStyleSheet(tokens, osColorSchemes)]);
```

Top level await will work if you update the file to `.mjs`

Reply:
Tried this and got more errors about importing modules. Gonna keep for now.

Reply:
Oh because the mixture of `.mjs` and `.cjs`. That's fine we can circle back 👍